cmd/agent/agent_test.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"github.com/stretchr/testify/assert"
+	"math/big"
+	"net"
+	"testing"
+	"time"
+)
+
+func TestKubeletCertificateNeedsRegeneration(t *testing.T) {
+
+	correctIPAddress := "167.71.188.113"
```

Comment:
Just for future reference, I like to use either well known ranges `192.168.0.0/16` or the `TEST-NET-1` example range `192.0.2.0/24`: https://stackoverflow.com/a/6778154/8976784